model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/latest/__init__.py

```diff
@@ -16,9 +16,8 @@
 from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.v1.tp_model import get_tp_model, generate_tp_model, \
     get_op_quantization_configs
 if FOUND_TF:
-    from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.v1.tpc_keras import get_keras_tpc as get_keras_tpc_latest
-    from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.v1.tpc_keras import generate_keras_tpc
+    from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.v1.tp_model import get_tp_model as get_keras_tpc_latest
```

Comment:
I'm aware that this is not the best way to expose this APIs, but note that this change is done in all /latest/ tpc init files, to support the current default API and testing.